CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format of this changelog is based on
 
 - Wraps colors around `\[` and `\]` for colors in `$PS1`. See
   https://unix.stackexchange.com/a/157884
+- Update text for autogenerated files
```

Comment:
should be under the unreleased section.

Reply:
![Learning-to-read-scaled](https://github.com/jmlopez-rod/m/assets/62945622/b134c516-4e9d-4872-8196-456607eabc7d)